src/wtc-utility-helpers.js

```diff
@@ -1,95 +1,71 @@
-var utilities = {};
-
 /**
  * Generate a random float number max and min.
- * 
+ *
  * ```javascript
  * utilities.floatRandomBetween(-10, 20); // 12.513
  * ```
- * 
- * @public
+ *
  * @param {number} min Minimum value.
  * @param {number} max Maximum value.
  * @return {number} Random number.
  */
-utilities.floatRandomBetween = function(min, max) {
+export const floatRandomBetween = (min, max) => {
   return Math.random() * (max - min + 1) + min;
-}
+};
 
 /**
  * Generate a random integer number max and min.
- * 
+ *
  * ```javascript
  * utilities.randomBetween(-10, 20); // 12
  * ```
- * 
- * @public
+ *
  * @param {number} min Minimum value.
  * @param {number} max Maximum value.
  * @return {number} Random number.
  */
-utilities.randomBetween = function(min, max){
+export const randomBetween = (min, max) => {
   return Math.floor(utilities.floatRandomBetween(min, max));
```

Comment:
We should update this to Math.round instead of Math.floor if we want this to be correct.

Reply:
Done!